src/logger/networking.c

```diff
@@ -72,7 +72,7 @@ void netw_main(double lat, double lon, double speed, double course)
 
     write(client_socket, unit_port, strlen(unit_port));
 
-    gen_json = generate_json(1, "ambulance", lat, lon, speed, course);
+    gen_json = generate_json_car(1, "ambulance", lat, lon, speed, course, 0, 0, 0, 0);
```

Comment:
As far as I see this code is disabled by "#if 0" statement
Why do you need to update here?
I guess this code inside #if 0 ... #endif might be removed at all - please check if all works without it and remove if all ok